src/Sulu/Component/Content/Tests/Functional/Mapper/ContentMapperTest.php

```diff
@@ -3410,6 +3411,7 @@ public function testLanguageCopyInternalLink()
         $data = [
             'title' => 'Page-1',
             'internal_link' => $page->getUuid(),
+            'nodeType' => RedirectType::INTERNAL,
         ];
 
         $data = $this->mapper->save($data, 'internal-link', 'sulu_io', 'de', 1);
```

Comment:
Yeah, they did, couldn't figure out what the reason was. Maybe I'll invest some more time here to find out in detail.